<launch>
  <!--Camera 1: cup orientation tracker-->
  <node type="color_detector_node" pkg="pr2_lfd_vision" name="color_detector_node_h">
    <remap from="color_middle_xy" to="color_middle_xy1" />
    <remap from="color_occupied_ratio" to="color_occupied_ratio1" />
    <param name="camera" value="1" />
    <param name="mode" value="2" />
    <param name="num_detectors" value="2" />
    <param name="color_files_base" value="$(find pr2_lfd_vision)/h" />
  </node>
</launch>
